test/.eslintrc

```diff
@@ -2,6 +2,8 @@
 
 rules:
   ## common module is mandatory in tests
-  required-modules: [2, common]
+  no-var: 2
   prefer-assert-iferror: 2
   prefer-assert-methods: 2
+  prefer-const: 2
+  required-modules: [2, common]
```

Comment:
Keep the comment in line 4 right above this rule, as this is the relevant line for that comment.

I won't hold up landing this over it, but I'd recommend formatting this `.eslintrc` file like the main one which is to categorize the rules as they appear in the ESLint docs, and alphabetically order them inside each category (which is how they are ordered in the docs).

So, it would be something like this:

```
## Test-specific linter rules

rules:
  # ECMAScript 6
  # http://eslint.org/docs/rules/#ecmascript-6
  no-var: 2
  prefer-const: 2

  # Custom rules in tools/eslint-rules
  prefer-assert-iferror: 2
  prefer-assert-methods: 2
  ## common module is mandatory in tests
  required-modules: [2, common]
```